data/allPossibleCategories.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function() {
+  const allPossibleCategoriesRaw = ['Appearance', 'Developer Tools', 'Productivity', 'Themes', 'Integrations', 'Viewer', 'Search', 'Tags', 'Editor', 'Files', 'Personal Knowledge Management'];
```

Comment:
Can't have this duplicate list here

Reply:
I've raise a PR in the main codebase to maintain this list in @joplin/lib, I'll include @joplin/lib to get this list after that PR is merged.